forem/forem
# Removed the autoprefixer-rails deprecated gem

## What type of PR is this? (check all applicable)


## Description

This removes a gem that was modifying CSS assets to add vendor prefixes where necessary. We still need to use postcss/autoprefixer, but we do it via an npm script which runs as a rake task post assets precompile.

## Related Tickets & Documents

#9965

## QA Instructions, Screenshots, Recordings

As it potentially affects all CSS files, a smoke test of the site is required. Navigate to pages and observe if there is anything that stands out in regards to the look of the site.

Do this for FireFox and Safari as well, not just Chromium based browsers.

I've done this myself already and things look fine.

## Added tests?


## Added to documentation?


## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![Jimmy Fallon saying "Brace yourself."](https://media.giphy.com/media/11JeH5lk2G494A/giphy.gif)
